src/site/content/en/blog/how-to-use-local-https/index.md

```diff
@@ -0,0 +1,317 @@
+---
+title: How to use HTTPS for local development
+subhead: Sometimes, you need to run your local development site with HTTPS. Tools and tips to do this safely and quickly.
+authors:
+  - maudn
+date: 2020-12-07
+updated: 2020-12-07
+hero: hero.jpg
+tags:
+  - blog
+  - security
+---
+
+<hr/>
+
+**Outline:**
+
+- [Running your site locally with HTTPS using mkcert (recommended)](<#running-your-site-locally-with-https-using-mkcert-(recommended)>)
+- [Running your site locally with HTTPS: other options](#running-your-site-locally-with-https:-other-options)
+
+<hr/>
+
+_In this article, statements about `localhost` are valid for `127.0.0.1` and `[::1]` as well, since they both describe the local computer address, also called "loopback address". Also, to keep things simple, the port number isn't specified._
+_So when you see `http://localhost`, read it as `http://localhost:{port}` or `http://127.0.0.1:{port}`._
+
+If your production website uses HTTPS, you want your local development site to behave **like an
+HTTPS site** (if your production website doesn't use HTTPS, [make it a priority to switch to HTTPS](/why-https-matters/)).
+Most of the time, you can trust `http://localhost` to behave **like an HTTPS site**. But [in some
+cases](/when-to-use-local-https), you need to run your site locally with HTTPS. **Let's take a look
+at how to do this.**
+
+**⏩ Are you looking for quick instructions, or have you been here before?
+Skip to the [Cheatsheet](/#using-mkcert:-cheatsheet).**
+
+## Running your site locally with HTTPS using mkcert (recommended)
+
+To use HTTPS with your local development site and access `https://localhost` or
+`https://mysite.example` (custom hostname), you need a [TLS
+certificate](https://en.wikipedia.org/wiki/Public_key_certificate#TLS/SSL_server_certificate). But
+browsers won't consider just any certificate valid: your certificate needs to be **signed** by an entity that
+is trusted by your browser, called a trusted **[certificate authority (CA)](https://en.wikipedia.org/wiki/Certificate_authority)**.
+
+So, what you can do is create a certificate and sign it with a CA that is **trusted locally** by your
+device and browser.
+
+One tool that helps you do this in a few commands is
+[mkcert](https://github.com/FiloSottile/mkcert). Here is how it works:
```

Comment:
```suggestion
To create a certificate and sign it with a CA that is **trusted locally** by your device 
and browser, use [mkcert](https://github.com/FiloSottile/mkcert). Here is how it works:
```

Reply:
To me the rationale of using mkcert wasn't obvious at first, so my idea with "So, what you can do... [...] One tool..." is to explain this step-by-step to the reader. I want to make sure they understand the split between the solution (use a local CA) and the tool that enables this solution (mkcert _for example_).
And I'm afraid the suggested formulation is too much at once.
But I'm open to suggestions that enable to bring in the concept gradually!

Reply:
Edited back to your original intention, just slightly rephrased for brevity and flow.

Reply:
Looking great, thanks!